packages/editor-ui/src/components/ParameterInputList.vue

```diff
@@ -131,12 +130,30 @@ export default mixins(
 				return this.$store.getters.activeNode;
 			},
 			indexToShowSlotAt (): number {
-				if (this.node.type === WEBHOOK_NODE_TYPE) return 1;
+				let index = 0;
+				const credentialsDependencies = this.getCredentialsDependencies();
 
-				return 0;
+				this.filteredParameters.forEach((prop, propIndex) => {
+					if (credentialsDependencies.has(prop.name)) {
+						index = propIndex + 1;
+					}
+				});
+
+				return index < this.filteredParameters.length ?
+					index : this.filteredParameters.length - 1;
 			},
 		},
 		methods: {
+			getCredentialsDependencies() {
+				const dependencies = new Set();
+				const nodeType = this.$store.getters.nodeType(this.node.type, this.node.typeVersion);
```

Comment:
need to address unknown types here
```suggestion
				const nodeType = this.$store.getters.nodeType(this.node.type, this.node.typeVersion) as INodeType | undefined;
```
<img width="1188" alt="Bildschirmfoto 2022-06-07 um 14 09 04" src="https://user-images.githubusercontent.com/4711238/172375466-129cfbee-27dd-4b19-8938-656c2028bd2d.png">

Reply:
also breaking nodes without creds